src/System.Windows.Forms/tests/UnitTests/HScrollBarAccessibleObjectTests.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace System.Windows.Forms.Tests
+{
+    public class HScrollBarAccessibleObjectTests : IClassFixture<ThreadExceptionFixture>
```

Comment:
https://github.com/dotnet/winforms/blob/master/Documentation/testing.md#naming
```suggestion
    public class HScrollBar_HScrollBarAccessibleObjectTests : IClassFixture<ThreadExceptionFixture>
```